### PR TITLE
chore(e2e): create test to ensure `--fcc-expected--` does not show up in the hint text

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This adds a simple test to ensure that `--fcc-expected--` never actually shows up as hint text for challenge. For our purposes, I'm using Step 4 of the CSS Penguin Project. 

Related to https://github.com/freeCodeCamp/freeCodeCamp/pull/56577

### CODE REVIEW

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Thanks @a2937 I think this will help. Could you make it check for a specific hint? It's a little more reliable to test for the presence of something than the absence. Reason being an empty string would pass, while not being what we want.

**Comment by a2937** [comment]
It's funny you should say that. Cause I'm getting something really weird happening here. In the automated tests, the element has no value. But in the browser it's completely fine.

**Comment by a2937** [comment]
My best bet is that `actual` is not being assigned a value. So either the error message is missing a value or the value of `actual` is reassigned to a none value.

File: e2e/lower-jaw.spec.ts [hunk 1]
```diff
@@ -195,3 +216,30 @@ test('should display the text of submit and go to next challenge button accordin
...
+  page,
+  isMobile,
+  browserName
+}) => {
+  await page.goto(
+    'learn/2022/responsive-web-design/learn-css-transforms-by-building-a-penguin/step-4'
+  );
+  const editor = getEditors(page);
+  const checkButton = page.getByRole('button', { name: 'Check Your Code' });
+  await checkButton.click();
```
Comment (ojeytonwilliams) [inline e2e/lower-jaw.spec.ts:230]:
```suggestion
```
This needs to be done after the `fill`

File: e2e/lower-jaw.spec.ts [hunk 2]
```diff
@@ -195,3 +216,30 @@ test('should display the text of submit and go to next challenge button accordin
...
+  await focusEditor({ page, isMobile });
+  await clearEditor({ page, browserName });
+
+  await editor.fill(
+    'body{background:linear-gradient(45deg, rgb(118, 201, 255), rgb(247, 255, 222));margin:0;padding:0;width:5%;height:100vh}'
+  );
+
```
Comment (ojeytonwilliams) [inline e2e/lower-jaw.spec.ts:237]:
```suggestion
  await checkButton.click();
  
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
This should get it working in CI:

**Review by ojeytonwilliams — APPROVED** [review]
The code looks solid. Assuming no surprises in CI, LGTM :+1:

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**